src/planner/mod.rs

```diff
@@ -301,6 +301,8 @@ pub enum PlannerError {
     NixOs,
     #[error("`nix` is already a valid command, so it is installed")]
     NixExists,
+    #[error("WSL1 is not supported, please upgrade to WSL2")]
```

Comment:
Think we can give them a link how to?

Reply:
```suggestion
    #[error("WSL1 is not supported, please upgrade to WSL2: https://learn.microsoft.com/en-us/windows/wsl/install#upgrade-version-from-wsl-1-to-wsl-2")]
```

Just gotta hope MS doesn't move that link...